_includes/_global_js.html

```diff
@@ -0,0 +1,55 @@
+<!-- This should be inserted in the page *before* any page-specific JS -->
+<script src="/assets/js/jquery.js"></script>
+<script src="/assets/js/bootstrap.min.js"></script>
+{% for js_name in page.extra_js %}<script src="/assets/js/{{ js_name }}"></script>{% endfor %}
+
+<script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+  ga('create', 'UA-77346638-1', 'codefordc.org');
+  ga('set', 'forceSSL', true);
+  ga('set', 'anonymizeIp', true);
+  ga('send', 'pageview');
+
+</script>
+
+<script>
+    //highlights current page in sitewide nav (_navbar.html)
+    $(document).ready(function(){
+      $("#navbar-nav a").each(function() {
+        if (this.href == window.location.href ||
+            this.href + "index.html" == window.location.href ||
+            this.href == window.location.href + "index.html" ) {
+
+            $(this).parent().addClass("active");
+
+          }
+        });
+    });
```

Comment:
Local testing it doesn't look like this works/is used. I'd prefer to just delete this snippet.